### ISSUE TITLE
PluginSidebar not enqueued in Windows environment

### ISSUE THREAD

**Issue by sgroleau-mishmash** [alleyinteractive/apple-news#920]
### Issue description
Starting from version 2.2.0 up to the current latest version 2.3.1, the PluginSidebar (Apple News Options) is not being displayed anymore when editing posts. Reverting to version 2.1.3 fixes the issue.

No matter the version, we can always see the Section Mappings page and refresh the list, meaning channel config/credentials are OK.

It must not be a widespread issue, but something specific to our setup.
@kevinfodness, would love your help figuring out what change between 2.1.3 and 2.2.0 could cause this behavior.

### Screenshots
- Using version 2.1.3
![publish_to_apple_news_213](https://user-images.githubusercontent.com/56895212/148592667-cd4a9078-e718-4a11-af5f-dace48b238b0.png)
- Using version 2.2.0 or greater
![publish_to_apple_news_220](https://user-images.githubusercontent.com/56895212/148592738-03b62302-e97c-45b7-b06c-974ea6e14b9b.png)

### Testing environment
Local server on Windows 10 using WAMP, tested on 2 PCs with different WAMP setups
- PHP 7.4.9 and MySQL 5.7.31
- PHP 7.4.26 and MySQL 8.0.27

WordPress 5.8.3
- Multisite feature (Publish to Apple News is activated on blog 1 only)
- Gutenberg editor

### What we've tried
- Changing WordPress versions: 5.7.5, 5.8.2 and 5.8.3
- Switching WordPress language: English and French
- Replacing our custom theme with Twenty Twenty-One
- Disabling all other plugins
- Resaving Publish to Apple News plugin settings
- Using different browsers: Chrome 96 and Firefox 95

**Comment by kevinfodness** [comment]
You should be able to click the three dots to the right of the gear icon and it should be visible in that list. If you click it there it should come back on the toolbar. If it does not appear there, then there might be a JavaScript error that's preventing it from being registered. Could you open the developer tools in your browser, click on Console, and see if there is an error that appears to be related to Publish to Apple News plugin code? If so, could you provide it here, please?

**Comment by sgroleau-mishmash** [comment]
We finally took some time to investigate this issue since we wanted to update the plugin version in production. The issue occurs on Windows machines, so it was only a problem when testing locally.

Since [this commit](https://github.com/alleyinteractive/apple-news/commit/58303f73243d03432e259e94a51a2f45029c3b6c#diff-e8f27f50f3b75ae544f31c2c85e692a52eaf636bd02f02dd89fd7e9d78a472b6), pluginSidebar dependencies are loaded from a file, which is validated using WordPress' [`validate_file()`](https://developer.wordpress.org/reference/functions/validate_file/). This function returns a value of `2` when the file path contains a Windows drive path and your code returns without enqueuing pluginSidebar.js when a value other than `0` is received.

We removed the `validate_file()` check and the pluginSidebar was now visible. We were able to push articles to our sandbox channel while testing locally on Windows.

Other plugins had the same issue and the fix was either a custom directory traversal check or changing `0 !== validate_file( $dependency_file )` to `1 === validate_file( $dependency_file )`.
